### PR TITLE
Release v2.1.0-rc.5

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve? 
Prepare release.

### What is changed and how it works?
Add changelog.

### Check List 

Tests 
 - Unit test

### CODE REVIEW

**Comment by disksing** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 tidb-test=release-2.1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # PD Change Log
 
+## v2.1.0-rc.5
+- Fix the issues related to `pd-ctl` reading the Region key [#1298](https://github.com/pingcap/pd/pull/1298) [#1299](https://github.com/pingcap/pd/pull/1299) [#1308](https://github.com/pingcap/pd/pull/1308)
+- Fix the issue that the `regions/check` API returns the wrong result [#1311](https://github.com/pingcap/pd/pull/1311)
+- Fix the issue that PD cannot restart join after a PD join failure [1279](https://github.com/pingcap/pd/pull/1279)
```
Comment (nolouch) [inline CHANGELOG.md]: s/1279/#1279

**Comment by disksing** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 tidb-test=release-2.1

**Pull request merged**